api/proto/config/storage.proto

```diff
@@ -148,6 +149,12 @@ message ContentTree {
   // instance. In case of new instance using the same content tree, EVE will get
   // new content tree UUID in the device configuration from the controller.
   int64 generation_count = 9;
+
+  // An arbitrary string. Usage depends on the content purpose. Currently the
+  // only usage is with custom application blobs (AppCustom) - this string will
+  // appear in the restAPI endpoint available at http://169.254.169.254. So user
+  // application can read that data and treat the blob accordingly.
+  string customMetaData = 10;
```

Comment:
Can it contain any sensitive information (and be wrapped in a CipherBlock)?

Reply:
I can't think of such scenario. Usecase we had in mind is just a version string